### DISCUSSION TITLE
fa/manager/configuration-and-advanced-settings/How-to-configure-Hiddify-panel-properly/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4332] [Content Help کمک به اصلاح محتوا]
# fa/manager/configuration-and-advanced-settings/How-to-configure-Hiddify-panel-properly/

The proxy manager platform.

https://hiddify.com/fa/manager/configuration-and-advanced-settings/How-to-configure-Hiddify-panel-properly/

**Comment by Sajjad-Arjmand** [comment]
در تنظیمات همچین چیزی نداریم اصلا:
امکان اتصال به پنل مرکزی (مالتی سرور) وجود دارد. پس از اتصال به پنل مرمزی، می‌توان کاربران را به صورت یکپارچه روی پنل مرکزی تعریف نمود و میزان استفاده همه سرورها به صورت تجمعی اضافه ‌شود.
